portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,16 +51,25 @@ public void init() {
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get the input from the form.
-    String name = getParameter(request, "name-input", "");
+    String author = getParameter(request, "name-input", "");
     String message = getParameter(request, "message-input", "");
-    //Add new data to data structure
-    comments.add(new Comment(name, message));
+    //Taking system time for keepijg track of timestamps of comments
```

Comment:
I think you could make the variable name more clear about its purpose and drop the comment.

If not, there's a typo in here ;)